.gitignore

```diff
@@ -3,3 +3,4 @@ public/*
 resources/*
 *~
 .hugo_build.lock
+content/projects/_index.md
```

Comment:
I think this should be left out of .gitignore, the file 'content/projects/_index.md' is in the git repo and should remain there.